docs/UserGuide.md

```diff
@@ -133,7 +133,13 @@ This is a sample screenshot of what you can expect from the `help` command.
 Format: `help`
 
 ### Importing Excel sheet
-Add a list of students from an Excel file into Tracey.
+Add a list of students from an Excel file into Tracey. Import feature can be used in the command line or import button.
+
+Format: `import FILE PATH`
+
+Examples of usage:
+* `import C:\Users\Documents\tp.xlsx`
+* `import C:\Users\Documents\Desktop\tp\students.xlsx`
```

Comment:
I think can give more details on the file path
eg. the file path is relative to the root directory

Reply:
Good suggestion! will add it